joblib/test/test_parallel.py

```diff
@@ -40,15 +40,18 @@
     from Queue import Queue
 
 
-from joblib.parallel import Parallel, delayed, SafeFunction, WorkerInterrupt
-from joblib.parallel import mp, cpu_count, VALID_BACKENDS
+from joblib._parallel_backends import (SequentialBackend, SafeFunction,
+                                       WorkerInterrupt)
+from joblib.parallel import (Parallel, delayed, register_parallel_backend)
+
+from joblib.parallel import mp, cpu_count, VALID_BACKENDS, effective_n_jobs
 from joblib.my_exceptions import JoblibException
 
 import nose
 from nose.tools import assert_equal, assert_true, assert_false, assert_raises
 
 
-ALL_VALID_BACKENDS = [None] + VALID_BACKENDS
+ALL_VALID_BACKENDS = [None] + list(VALID_BACKENDS.keys())
```

Comment:
I think that I would like that list to be sorted, as in "sorted(VALID_BACKENDS.keys())", just for the sake of reproducibility.